sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/Properties/AssemblyInfo.cs

```diff
@@ -9,4 +10,6 @@
                               "012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265" +
                               "e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593d" +
                               "aa7b11b4")]
-[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2, PublicKey=0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7")]
\ No newline at end of file
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2, PublicKey=0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7")]
+
+[assembly: SuppressMessage("Maintainability", "CA1507:Use nameof to express symbol names", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.Azure.WebJobs.Host.Converters.CompositeObjectToTypeConverter`1.#ctor(System.Collections.Generic.IEnumerable{Microsoft.Azure.WebJobs.Host.Converters.IObjectToTypeConverter{`0}})")]
```

Comment:
Why is this needed? We also use sources dependency in other extensions but don't have this suppression.

Reply:
You ignored the same error globally https://github.com/Azure/azure-sdk-for-net/blob/7318c81564c6e70b7b8c70c77d80adc0dbd70203/sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Microsoft.Azure.WebJobs.Extensions.ServiceBus.csproj#L9

Reply:
Yes, yes I did.

Reply:
Mind updating that here?